### ISSUE TITLE
demon skin 

### ISSUE THREAD

**Issue by dybbukk** [robert388/Necrosis-classic#144]
hi there, not sure what happened, was working fine couple days ago, but now Demon Skin isn't showing up as an option to cast when clicking necrosis., tried to unisntall and reinstall, no luck

**Comment by dybbukk** [comment]
and soulstone doesnt work either

**Comment by SuperBigouu** [comment]
I think it has to do with ranks, i opened a ticket a few days ago as well.
Everything worked fine, but now that i have max rank demon skin / SS / HS the buttons don't light up when you have them or even work when you clic them

**Comment by dybbukk** [comment]
yeah the spells on cast has now been fixed, but the ss/hs issue persists - which is weird, because the ss was working find even just last week
